requirements.txt

```diff
@@ -19,6 +19,7 @@ jsonschema
 marshmallow
 marshmallow-jsonschema
 marshmallow-dataclass
+pytkdocs
```

Comment:
Is this needed at runtime? I thought this was only used to generate the compiled docs json.

Reply:
It's needed for easily loading back the JSON, see `restloader` and `googleloader` in `marshmallow_schema_utils`.

Reply:
Also it's used in `extract_schema.py`, so hypothetically if we didn't need it at runtime should I just remove it as a requirement and assume that someone running that script installs it separately?

Reply:
removed